apps/src/lib/kits/maker/toolkit.js

```diff
@@ -0,0 +1,118 @@
+/*
+ * @file Static interface to Maker Toolkit, to simplify App Lab code interfacing
+ * with maker and provide clean setup/cancel/reset patterns.
+ */
```

Comment:
Okay @davidsbailey, PTAL.

In the last couple of commits I've pulled a lot of the behavior that was in applab.js up into this maker/toolkit.js file, which I'm explicitly using as the one bit of Maker toolkit that understands its static/global context and knows that only one board should be connected at any time.  It makes for a much nicer interface for applab code, and puts everything I need for cancelling a connection in progress in one place.